src/cmd.ts

```diff
@@ -153,6 +153,27 @@ function renderPage(
   return ejs.render(layout.contents, context);
 }
 
+function renderPageContentsForRSS(
+  page: PageType,
+  appBaseURL: string,
+) {
+  const renderer =  new marked.Renderer();
+  renderer.link = (href, title, text) => {
+    if (href && href.startsWith("/")) {
+      return `<a href="${appBaseURL}${href}">${text}</a>`;
```

Comment:
Minor nit: It might be nice to have a function to join appBaseURL and its supplied path to ensure this is robust and reusable. That might help make some of this code more concise.

If you choose to do that, I'd consider using the [URL](https://nodejs.org/api/url.html) module to construct it as a matter of good practice.

Reply:
I can make that change. Good call, since I'm doing this in several places.